meetings/2023-06-21.md

```diff
@@ -0,0 +1,130 @@
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Pierre-Antoine
+
+### Introductions
+* name: text
+
+---
+
+
+## Topics
+
+### Recording demos
+* SC: I don't mind recording it but this raises the question of where to host the video afterward. I could put it on my website but the CG probably does not want to rely on my website being up -- and I don't want this responsibility.
+
+### WIP Implementation Feedback
+* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.
+* eP: IIUC we are about to accept the WebID section of the charter, I just setup my WebID. ~6 lines of nginx configuration. Based on [Cool URIs for the Semantic Web - 4.3. 303 URIs forwarding to Different Documents](https://www.w3.org/TR/cooluris/#r303uri) . Simple deployment was super fast and easy.
+* SC: would you consider the Turtle / JSON-LD and the HTML representation to be equivalent?
+* eP: not really. They both describe me, but intended to different audiences.
+* PA: Have you tried using this? I have experimented recently to have a pod on my CSS that is not using /profile/card#me but rather that I've been using some URL for some URL. I've tried it something on that page to be Solid compliant. WebID which is not hosted on a pod even though it contains the required properties, to find my pod and access. Looking at your, it redirects?
+* eP: It is just a static file currently. I hope to have an answer to your question next week. I can file bug reports about where the WebID is hosted.
+* JZ: about profiles not stored in a Pod: that's what ESS does currently does. And this cause problems in the WebID profile group. We are interested in your feedback.
+* PA: happy to join. My reading of the WebID profile spec was that it was ok, but let's discuss this.
```

Comment:
From perspective of expired WebID WG, it shouldn't matter where a WebID Profile is hosted — in a pod, in a filesystem, on a non-Solid (HTTPS, SFTP, etc.) server, etc. (Originally, HTTP, FTP, and the like were OK, but since security has been more of a concern, HTTPS or other secured (typically by TLS) protocols are generally a SHOULD if not a MUST.) It's just necessary that the WebID dereference to the Profile document, which contains the WebID again and one or more associated public keys.

In other words — if any Solid server chokes on a WebID Profile because it's not hosted on a Solid server, that's a bug in the Solid server (or perhaps in the non-Solid server hosting the Profile document).

Reply:
There is more to what's minuted here and perhaps the context wasn't entirely clear to everyone either. There is indeed the general read aspect of WebID Profiles (as per WebID 1.0) - which is what you're referring to - and then there is the more specific read/write aspect of Solid WebID Profile. Obviously we can't claim interoperability between Solid applications and arbitrary HTTP servers with respect to modifying the (Solid) WebID Profile Document. The Solid WebID Profile panel/team/force is trying to clarify this stuff.